x-pack/plugins/discover_enhanced/public/plugin.ts

```diff
@@ -15,25 +15,11 @@ import { KibanaLegacySetup, KibanaLegacyStart } from '../../../../src/plugins/ki
 import {
   EmbeddableSetup,
   EmbeddableStart,
-  EmbeddableContext,
   CONTEXT_MENU_TRIGGER,
 } from '../../../../src/plugins/embeddable/public';
-import {
-  ExploreDataContextMenuAction,
-  ExploreDataChartAction,
-  ACTION_EXPLORE_DATA,
-  ACTION_EXPLORE_DATA_CHART,
-  ExploreDataChartActionContext,
-} from './actions';
+import { ExploreDataContextMenuAction, ExploreDataChartAction } from './actions';
```

Comment:
It's doesn't seem to be used at the moment, but these types should be exported from the top level of the plugin so others can still references them, right? In the old version they were put into the `ActionContextMapping`, but without that there's no way to get them